package.json

```diff
@@ -23,7 +23,7 @@
     "@mate-academy/bemlint": "^0.1.1",
     "@mate-academy/eslint-config": "*",
     "@mate-academy/linthtml-config": "0.0.2",
-    "@mate-academy/scripts": "^0.9.1",
+    "@mate-academy/scripts": "^1.2.6",
     "@mate-academy/stylelint-config": "*",
     "backstopjs": "^5.0.1",
     "eslint": "^5.16.0",
```

Comment:
You don't need to push this file. Push only src folder and readme.md

Reply:
Yes, I knew it